.devcontainer/bin/00-bootstrap.sh

```
@@ -0,0 +1,27 @@
+# Ensure the script is run from the correct directory
+cd "$(dirname "$0")/../.."  
+echo "Preparing toolchains.."
+echo "Checking for Claude Code CLI"
+
+#check if latest claude-code is installed
+if ! command -v claude &> /dev/null; then
+    echo "Claude CLI not found, installing..."
+    npm install -g @anthropic-ai/claude-code
+
+else
+    echo "Claude CLI installed in "`which claude`", checking version...one moment please"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)